.github/workflows/rust.yml

```diff
@@ -230,7 +230,7 @@ jobs:
         run: docker-compose up -d influxdb minio redis
 
       - name: Run cargo-tarpaulin
-        uses: actions-rs/tarpaulin@v0.1.2
+        uses: actions-rs/tarpaulin@v0.1
```

Comment:
so this is equivalent to saying "the latest 0.1.x" version?

Reply:
yes 😊